src/aqua/spell/spell_service.aqua

```diff
@@ -107,3 +120,5 @@ service Spell:
   set_trigger_config(config: TriggerConfig) -> UnitValue
   set_u32(key: string, value: u32) -> UnitValue
   store_error(error: LastError, error_idx: u32, particle_timestamp: u64) -> UnitValue
+  store_log(log: string) -> UnitValue
```

Comment:
maybe it should also accept a prefix? 🤔 or even array of prefixes

eg for deal_id